app/controllers/api/v2/webhooks_controller.rb

```diff
@@ -57,6 +57,11 @@ def update
       def destroy
         process_response @webhook.destroy
       end
+
+      api :GET, '/webhooks/events', N_('List available events for subscription')
```

Comment:
Added new API endpoint to get the available events. Mainly for UI, because in API/Hammer all available events should be shown in API docs/command line help.